drivers/sensors/mpu60x0.c

```diff
@@ -884,6 +995,104 @@ static off_t mpu_seek(FAR struct file *filep, off_t offset, int whence)
   return 0;
 }
 
+/****************************************************************************
+ * Name: mpu60x0_ioctl
+ ****************************************************************************/
+
+static int mpu_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
+{
+  FAR struct inode *inode = filep->f_inode;
+  FAR struct mpu_dev_s *priv = inode->i_private;
+  uint8_t write_data = (uint8_t)arg;
+  int ret = OK;
+
+  switch (cmd)
+    {
+      /* Sets the accelerometer full scale range. Arg: uin8_t value */
+
+      case SNIOC_SET_AFS_SEL:
+        ret = __mpu_write_accel_config(priv, write_data);
+        priv->afs_sel = write_data;
```

Comment:
should we do this if above call fails?

Reply:
Good. I'll not modify priv->afs_sel if it fails.